.github/workflows/release.yaml

```diff
@@ -30,3 +30,19 @@ jobs:
           # Ensure target branch for release is "master"
           commit: master
           token: ${{ secrets.GITHUB_TOKEN }}
+  update-version:
```

Comment:
I'm not sure this will work as you'd like it to, unless you plan to always use package version `master`.

Reply:
We do plan to only use `master` for the package